New package: GridapTopOpt v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: GridapTopOpt
- Repository: https://github.com/zjwegert/GridapTopOpt.jl
- Created by: @zjwegert
- Version: v0.1.0
- Commit: a13ef80fea13bd6b2e14b83b787e73b6729d4cd6
- Reviewed by: @zjwegert
- Reference: https://github.com/zjwegert/GridapTopOpt.jl/commit/a13ef80fea13bd6b2e14b83b787e73b6729d4cd6#commitcomment-142088438
- Description: A computational toolbox for level set-based topology optimisation in Julia
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6eafac106c8e32c9b4e9feafbe64daf4c93ba8754c3a63dbe22452a7d13f594e0ccd0204c55e87126ee56186814ea2300021953218074ef3f8793fab1f76681fc60c508e0d36d0b27c4f41995dd95c11c3be9a7d068e4ff18ce2f225b7e57f18a64f632ba557cc762015f3476a537a66e87566efd235215a588c52dbeabe95325f90e914046008344ccd3f72529f016851898a12b0088c0a95823f70e662e529c16b11df06740400a954ae6a6b20194bc -->

goerz commented:
Could you rename this to `GridapTopologyOpt`?

JordiManyer commented:
> Could you rename this to `GridapTopologyOpt`?

Hi @goerz , any particular reason why? I'm trying to understand what could be bad about the name. Thanks!

zjwegert commented:
Hi @goerz, after conferring with our collaborators we have concluded that TopOpt is a common name. There is `TopOpt.jl` developed by Julia computing, a TopOpt Julia slack channel, and a JuliaTopOpt organisation [https://github.com/JuliaTopOpt](https://github.com/JuliaTopOpt). Googling `topopt` also suggests that this is a well accepted abbreviation.

goerz commented:
[noblock] Fair enough! With that context, I withdraw my objections.

Not that I'm super happy about that context existing in the first place, but I suppose that entire "TopOpt" ecosystem developed fairly early, when the registration of new Julia packages was still mostly unreviewed. But now that it exists, I agree that new packages connecting to that ecosystem have a good reason to reference the existing name.

@zjwegert and @JordiManyer: Please note that you have to unblock your own comments as well for the registration to proceed.

zjwegert commented:
[noblock] 

Thanks @goerz! 👍

JuliaRegistrator committed:
New package: GridapTopOpt v0.1.0

UUID: 27dd0110-1916-4fd6-8b4b-1bc109db1170
Repo: https://github.com/zjwegert/GridapTopOpt.jl.git
Tree: d17f4db908b8bd01d17914b699a0568de3b19793

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
